loadtest/control/simplecontroller/actions.go

```diff
@@ -41,10 +41,12 @@ func (c *SimpleController) login() control.UserStatus {
 		return c.newErrorStatus(err)
 	}
 
-	err = c.user.Connect()
-	if err != nil {
-		return c.newErrorStatus(err)
-	}
+	errChan := c.user.Connect()
+	go func() {
+		for err := range errChan {
+			c.status <- c.newErrorStatus(err)
```

Comment:
So we have to revamp the actions to send the error and info status to the channel, right? Instead of having two different ways to cope with status

Reply:
Not sure whether this was something for me or just an observation. :sweat_smile: We are just changing the `Connect` method to return the channel. The actions like signUp, login, logout remains the same because now the controller has the status channel as a field.

I am not sure if you meant for me to change something.

Reply:
Well is a question and the action to take is based on the answer :smile: . 

I mean, why do we need to return synchronous status in the controller actions if we are able to pass that status through the status channel and process it in the `handleStatus` function.

If we want to change the status logic inside the controller actions we have two options:
- Change all in this PR
- Create a new ticket to revamp it in other PR

Reply:
I'd opt for the second option. I think this would be out of scope for the current PR and also I would tend to wait on a similar change until we start using those statuses so we can figure out exactly what we need. But that's excellent feedback @ethervoid, thank you!

Reply:
Yes, it would be ideal if the status channel is passed all the way from top to the individual user methods. It is not good to spin up goroutines without having a way to gracefully shut them down. If we have 100 controllers logging in, there would be 100 goroutines, which is not really needed.

We need to get this fixed some time.